notification/app/notification/controllers/Main.scala

```diff
@@ -55,8 +56,11 @@ final class Main @Inject()(
     }
   }
 
-  def pushTopic(topic: Topic): Action[Notification] = AuthenticatedAction.async(BodyJson[Notification]) { request =>
-    val push = Push(request.body, Left(topic))
+  @deprecated("A push notification can be sent to multiple topics, this is for backward compatibility only", since = "07/12/2015")
+  def pushTopic(topic: Topic): Action[Notification] = pushTopics
+
+  def pushTopics: Action[Notification] = AuthenticatedAction.async(BodyJson[Notification]) { request =>
+    val push = Push(request.body, Left(request.body.topic))
```

Comment:
Field `.topic` should be pluralised in: https://github.com/guardian/mobile-n10n/blob/f2b89e81b5a1b32c08d4da604641410f0ef8f1bd/common/src/main/scala/models/Notification.scala#L15